plugins/flowdock/app/assets/javascripts/flowdock/application.js

```diff
@@ -1,112 +1,2 @@
 //= require jquery_mentions_input/jquery.elastic.source.js
 //= require jquery_mentions_input/jquery.mentionsInput.js
```

Comment:
don't think so ... they need to be precompiled / merged otherwise we end up
with extra js loading / duplicate loading ... so I'd rather not

On Mon, Jun 13, 2016 at 7:21 PM, Zehua Liu <notifications@github.com> wrote:

> In plugins/flowdock/app/assets/javascripts/flowdock/application.js
> <https://github.com/zendesk/samson/pull/1064#discussion_r66900326>:
>
> > @@ -1,112 +1,2 @@
> >  //= require jquery_mentions_input/jquery.elastic.source.js
> >  //= require jquery_mentions_input/jquery.mentionsInput.js
>
> Can we load these two guys in _notify_buddy_box.html.erb and get rid of
> this AngularJS file altogether?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1064/files/e876c7d763293036fd0708d1e6738939443ad7b4#r66900326>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAAsZ3nxyCQvfL5Adx_FBxBHnMj-iuO0ks5qLhA6gaJpZM4Iz2PM>
> .
>